File: Assets/AltUnityTester/AltUnityDriver/AltUnity/AltUnityObjectProperty.cs
public struct AltUnityObjectProperty
{ 
    public string Component;
    public string Property;
    public string Assembly;

    public AltUnityObjectProperty(string component = "", string property = "") :
        this(component, property, null) { }

    public AltUnityObjectProperty(string component, string property, string assembly)
    {
        Component = component;
        Property = property;
        Assembly = assembly;
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings/java/src/main/java/ro/altom/altunitytester/AltUnityObjectProperty.java
package ro.altom.altunitytester;

public class AltUnityObjectProperty {
    public String component;
    public String property;
    public String assembly;

    public AltUnityObjectProperty(String assemblyName,String componentName, String propertyName) {
        assembly=assemblyName;
        component = componentName;
        property = propertyName;
    }
}
